commands/build-image.ts

```diff
@@ -85,6 +87,12 @@ export function buildImage(dockerFileUri?: vscode.Uri) {
             let terminal: vscode.Terminal = vscode.window.createTerminal('Docker');
             terminal.sendText(`docker build -f ${uri.file} -t ${value} ${uri.path}`);
             terminal.show();
+            
+            if (reporter) {
```

Comment:
Is there any other scenario that `reported` wouldn't be defined, besides the `package.json` for the extension somehow getting deleted? Or is this guard catching something else? Could a command be called before the reporter has been initialized?